<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<includes>
    <include location="global" impldecl="in implementation">widgets/guiAPI/menu.h</include>
    <include location="global" impldecl="in implementation">widgets/guiAPI/menuitem.h</include>
</includes>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>514</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="bottomMargin">
     <number>12</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="lc::ui::api::Menu" name="menuDemo_Project">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="lc::ui::api::Menu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="lc::ui::api::Menu" name="menuSelect">
     <property name="title">
      <string>&amp;Select</string>
     </property>
     <addaction name="actionSelect_All"/>
     <addaction name="actionSelect_None"/>
     <addaction name="actionInvert_Selection"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
   </widget>
   <widget class="lc::ui::api::Menu" name="menuDemo">
    <property name="title">
     <string>C&amp;reate</string>
    </property>
    <widget class="lc::ui::api::Menu" name="menuLine">
     <property name="title">
      <string>&amp;Line</string>
     </property>
     <addaction name="action2_Point_Line"/>
     <addaction name="actionPoint_Angle_Length_Line" />
     <addaction name="actionHorizontalLine" />
     <addaction name="actionVerticalLine" />
     <addaction name="actionParallelLine" />
     <addaction name="actionOrthogonalLine" />
     <addaction name="separator"/>
     <addaction name="actionRectangle" />
     <addaction name="actionPolygonCenCor" />
     <addaction name="actionPolygonCenTan" />
     <addaction name="actionPolygonCorCor" />
    </widget>
    <widget class="lc::ui::api::Menu" name="menuCircle">
     <property name="title">
      <string>C&amp;ircle</string>
     </property>
     <addaction name="actionCenter_Radius"/>
     <addaction name="actionCenter_Diameter"/>
     <addaction name="separator"/>
     <addaction name="action2_Point_Circle"/>
     <addaction name="action3_Point_Circle_2"/>
     <addaction name="separator"/>
     <addaction name="actionTan_Tan_Radius"/>
     <addaction name="actionTan_Tan_Tan"/>
    </widget>
    <widget class="lc::ui::api::Menu" name="menuArc">
     <property name="title">
      <string>Arc</string>
     </property>
     <addaction name="action3_Point_Arc"/>
     <addaction name="separator"/>
     <addaction name="actionStart_Center_End_2"/>
     <addaction name="actionStart_Center_Angle_2"/>
     <addaction name="actionStart_Center_Length"/>
     <addaction name="separator"/>
     <addaction name="actionStart_End_Angle_3"/>
     <addaction name="actionStart_End_Direction_2"/>
     <addaction name="actionStart_End_Radius_2"/>
     <addaction name="separator"/>
     <addaction name="actionCenter_Start_End_2"/>
     <addaction name="actionCenter_Start_Angle_2"/>
     <addaction name="actionCenter_Start_Length_2"/>
     <addaction name="separator"/>
     <addaction name="actionContinue"/>
    </widget>
    <widget class="lc::ui::api::Menu" name="menuEllipse">
     <property name="title">
      <string>Ellipse</string>
     </property>
     <addaction name="actionEllipse_Axis"/>
     <addaction name="actionEllipse_Arc"/>
     <addaction name="actionEllipse_FociPoints"/>
     <addaction name="actionEllipse_4Points"/>
    </widget>
    <widget class="lc::ui::api::Menu" name="menuDimension">
     <property name="title">
      <string>&amp;Dimension</string>
     </property>
     <addaction name="actionOrdinate"/>
     <addaction name="actionLinear"/>
     <addaction name="actionAligned"/>
     <addaction name="actionANG3PT"/>
     <addaction name="actionANG2LN"/>
     <addaction name="actionRadius"/>
     <addaction name="actionDiameter"/>
    </widget>
    <addaction name="actionAdd_Random_Lines"/>
    <addaction name="actionAdd_Random_Circles"/>
    <addaction name="actionAdd_Random_Arc"/>
    <addaction name="actionClear_Undoable_Stack"/>
    <addaction name="separator"/>
    <addaction name="actionPolyline"/>
    <addaction name="actionSpline"/>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionAdd_Random_Lines">
   <property name="text">
    <string>&amp;Add Random Lines</string>
   </property>
  </action>
  <action name="actionAdd_Random_Circles">
   <property name="text">
    <string>Add &amp;Random Circles</string>
   </property>
  </action>
  <action name="actionAdd_Random_Arc">
   <property name="text">
    <string>Add Ra&amp;ndom Arc</string>
   </property>
  </action>
  <action name="actionClear_Undoable_Stack">
   <property name="text">
    <string>&amp;Clear Undoable Stack</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Sa&amp;ve As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>C&amp;opy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
  </action>
  <action name="actionSelect_None">
   <property name="text">
    <string>Select &amp;None</string>
   </property>
  </action>
  <action name="actionInvert_Selection">
   <property name="text">
    <string>&amp;Invert Selection</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionEllipse_Axis">
   <property name="text">
    <string>&amp;Ellipse</string>
   </property>
  </action>
  <action name="actionEllipse_Arc">
   <property name="text">
    <string>&amp;Ellipse Arc</string>
   </property>
  </action>
  <action name="actionEllipse_FociPoints">
      <property name="text">
          <string>&amp;Ellipse Foci Points</string>
      </property>
  </action>
  <action name="actionEllipse_4Points">
      <property name="text">
          <string>&amp;Ellipse 4 Points</string>
      </property>
  </action>
  <action name="actionPolyline">
   <property name="text">
    <string>&amp;Polyline</string>
   </property>
  </action>
  <action name="actionSpline">
   <property name="text">
    <string>&amp;Spline</string>
   </property>
  </action>
  <action name="action2_Point_Line">
   <property name="text">
    <string>&amp;2 Point Line</string>
   </property>
  </action>
  <action name="actionPoint_Angle_Length_Line">
   <property name="text">
    <string>&amp;Point, Angle, Length</string>
   </property>
  </action>
  <action name="actionHorizontalLine">
   <property name="text">
    <string>&amp;Horizontal Line</string>
   </property>
  </action>
  <action name="actionVerticalLine">
   <property name="text">
    <string>&amp;Vertical Line</string>
   </property>
  </action>
  <action name="actionParallelLine">
   <property name="text">
    <string>&amp;Parallel Line</string>
   </property>
  </action>
  <action name="actionOrthogonalLine">
   <property name="text">
    <string>&amp;Orthogonal Line</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>&amp;Rectangle</string>
   </property>
  </action>
  <action name="actionPolygonCenCor">
   <property name="text">
    <string>&amp;Polygon (Cen,Cor)</string>
   </property>
  </action>
  <action name="actionPolygonCenTan">
   <property name="text">
    <string>&amp;Polygon (Cen,Tan)</string>
   </property>
  </action>
  <action name="actionPolygonCorCor">
   <property name="text">
    <string>&amp;Polygon (Cor,Cor)</string>
   </property>
  </action>
  <action name="actionCenter_Radius">
   <property name="text">
    <string>&amp;Center, Radius</string>
   </property>
  </action>
  <action name="action3_Point_Circle">
   <property name="text">
    <string>3 Point Circle</string>
   </property>
  </action>
  <action name="action2_Point_circle">
   <property name="text">
    <string>2 Point circle</string>
   </property>
  </action>
  <action name="action3_Point_Arc">
   <property name="text">
    <string>&amp;3 Point Arc</string>
   </property>
  </action>
  <action name="actionStart_Center_End">
   <property name="text">
    <string>Start, Center, End</string>
   </property>
  </action>
  <action name="actionCenter_Start_End">
   <property name="text">
    <string>Center, Start, End</string>
   </property>
  </action>
  <action name="actionStart_Center_Angle">
   <property name="text">
    <string>Start, Center, Angle</string>
   </property>
  </action>
  <action name="actionCenter_Start_Angle">
   <property name="text">
    <string>Center, Start, Angle</string>
   </property>
  </action>
  <action name="actionStart_End_Angle">
   <property name="text">
    <string>Start, End, Angle</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start, Center, Length</string>
   </property>
  </action>
  <action name="actionCenter_Start_Length">
   <property name="text">
    <string>Center, Start, Length</string>
   </property>
  </action>
  <action name="actionStart_End_Angle_2">
   <property name="text">
    <string>Start, End, Angle</string>
   </property>
  </action>
  <action name="actionStart_End_Direction">
   <property name="text">
    <string>Start, End, Direction</string>
   </property>
  </action>
  <action name="actionStart_End_Radius">
   <property name="text">
    <string>Start, End, Radius</string>
   </property>
  </action>
  <action name="actionStart_Center_End_2">
   <property name="text">
    <string>&amp;Start, Center, End</string>
   </property>
  </action>
  <action name="actionStart_Center_Angle_2">
   <property name="text">
    <string>Start, Center, An&amp;gle</string>
   </property>
  </action>
  <action name="actionStart_Center_Length">
   <property name="text">
    <string>Start, Center, &amp;Length</string>
   </property>
  </action>
  <action name="actionStart_End_Angle_3">
   <property name="text">
    <string>Start, &amp;End, Angle</string>
   </property>
  </action>
  <action name="actionStart_End_Direction_2">
   <property name="text">
    <string>Start, End, &amp;Direction</string>
   </property>
  </action>
  <action name="actionStart_End_Radius_2">
   <property name="text">
    <string>Start, End, &amp;Radius</string>
   </property>
  </action>
  <action name="actionCenter_Start_End_2">
   <property name="text">
    <string>&amp;Center, Start, End</string>
   </property>
  </action>
  <action name="actionCenter_Start_Angle_2">
   <property name="text">
    <string>Ce&amp;nter, Start, Angle</string>
   </property>
  </action>
  <action name="actionCenter_Start_Length_2">
   <property name="text">
    <string>Center, Start, Lengt&amp;h</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="text">
    <string>C&amp;ontinue</string>
   </property>
  </action>
  <action name="actionCenter_Diameter">
   <property name="text">
    <string>Center, &amp;Diameter</string>
   </property>
  </action>
  <action name="action2_Point_Circle">
   <property name="text">
    <string>&amp;2 Point Circle</string>
   </property>
  </action>
  <action name="action3_Point_Circle_2">
   <property name="text">
    <string>&amp;3 Point Circle</string>
   </property>
  </action>
  <action name="actionTan_Tan_Radius">
   <property name="text">
    <string>&amp;Tan, Tan, Radius</string>
   </property>
  </action>
  <action name="actionTan_Tan_Tan">
   <property name="text">
    <string>Ta&amp;n, Tan, Tan</string>
   </property>
  </action>
  <action name="actionConstruction_Line">
   <property name="text">
    <string>&amp;Construction Line</string>
   </property>
  </action>
  <action name="actionOrdinate">
   <property name="text">
    <string>&amp;Ordinate</string>
   </property>
  </action>
  <action name="actionLinear">
   <property name="text">
    <string>&amp;Linear</string>
   </property>
  </action>
  <action name="actionAligned">
   <property name="text">
    <string>&amp;Aligned</string>
   </property>
  </action>
  <action name="actionANG3PT">
   <property name="text">
    <string>A&amp;NG3PT</string>
   </property>
  </action>
  <action name="actionANG2LN">
   <property name="text">
    <string>AN&amp;G2LN</string>
   </property>
  </action>
  <action name="actionRadius">
   <property name="text">
    <string>&amp;Radius</string>
   </property>
  </action>
  <action name="actionDiameter">
   <property name="text">
    <string>&amp;Diameter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
